src/notation/internal/notationactions.cpp

```diff
@@ -677,6 +677,66 @@ const ActionList NotationActions::m_actions = {
                ShortcutContext::NotationActive,
                QT_TRANSLATE_NOOP("action", "Navigator"),
                QT_TRANSLATE_NOOP("action", "Toggle 'Navigator'")
+               ),
+    ActionItem("toggle-palette",
+               ShortcutContext::NotationActive,
+               QT_TRANSLATE_NOOP("action", "Palettes"),
+               QT_TRANSLATE_NOOP("action", "Toggle 'Palettes'")
+               ),
+    ActionItem("toggle-instruments",
+               ShortcutContext::NotationActive,
+               QT_TRANSLATE_NOOP("action", "Instruments"),
+               QT_TRANSLATE_NOOP("action", "Toggle 'Instruments'")
+               ),
+    ActionItem("inspector",
+               ShortcutContext::NotationActive,
+               QT_TRANSLATE_NOOP("action", "Inspector"),
+               QT_TRANSLATE_NOOP("action", "Toggle 'Inspector'")
+               ),
+    ActionItem("toggle-statusbar",
+               ShortcutContext::NotationActive,
+               QT_TRANSLATE_NOOP("action", "Status Bar"),
+               QT_TRANSLATE_NOOP("action", "Toggle 'Status Bar'")
+               ),
+    ActionItem("toggle-noteinput",
+               ShortcutContext::NotationActive,
+               QT_TRANSLATE_NOOP("action", "Note Input"),
+               QT_TRANSLATE_NOOP("action", "Toggle 'Note Input' toolbar")
+               ),
+    ActionItem("toogle-notationtoolbar",
```

Comment:
Typo:

toggle

You should check in other places too

Reply:
thanks, fixed